### DISCUSSION TITLE
Add Support for Trisquel

### DISCUSSION THREAD

**Discussion by TechnologyClassroom** [antonym/netboot.xyz#640] [Ideas]
Would there be interest in adding Trisquel to netboot.xyz?  Trisquel is a fork of Ubuntu with all proprietary software removed and complies with [GNU's FSDG](https://www.gnu.org/distros/free-system-distribution-guidelines.html).

https://trisquel.info/

The net install images can be found at http://mirror.fsf.org/trisquel-images/ and start with the `trisquel-netinst_` string.

**Comment by jiderlesi** [comment]
I aslo think it's a good idea to add support for trisquel.
